ha/core/cluster/cluster_manager.py

```diff
@@ -324,132 +321,22 @@ def get_system_health(self, element: CLUSTER_ELEMENTS = CLUSTER_ELEMENTS.CLUSTER
         """
 
         try:
-            # Currently, only "id" supported in the variable arguments.
+            # Check if unsupported element status requested.
```

Comment:
cluster_manager should be a simple pass thru.
All of these checks should go to SystemHealthController.

Class SystemHealthController(SystemHealth):
    def get_status():
        # All validations here
        super().get_status()

Reply:
Done.